ext/net/lib.deno_net.d.ts

```diff
@@ -736,7 +817,39 @@ declare namespace Deno {
    * @experimental
    * @category Network
    */
-  export interface QuicReceiveStream extends ReadableStream<Uint8Array> {}
+  export interface QuicReceiveStream extends ReadableStream<Uint8Array> {
+    /**
+     * 62-bit stream ID, unique within this connection.
+     */
+    readonly id: bigint;
+  }
+
+  /**
+   * **UNSTABLE**: New API, yet to be vetted.
+   * Establishes a secure connection over QUIC using a hostname and port.  The
+   * cert file is optional and if not included Mozilla's root certificates will
+   * be used. See also https://github.com/ctz/webpki-roots for specifics.
+   *
+   * ```ts
+   * const caCert = await Deno.readTextFile("./certs/my_custom_root_CA.pem");
+   * const conn1 = await Deno.connectQuic({ hostname: "example.com", port: 443, alpnProtocols: ["h3"] });
+   * const conn2 = await Deno.connectQuic({ caCerts: [caCert], hostname: "example.com", port: 443, alpnProtocols: ["h3"] });
+   * ```
```

Comment:
Sigh, we should really address https://github.com/denoland/deno/issues/16160 so it wraps lines